### PR TITLE
Add Ramda as an available library

### PR DESCRIPTION
**Pull request by johnmichel**
This adds support for [Ramda](http://ramdajs.com/), another functional programming library.

### CODE REVIEW

**Comment by smt** [comment]
This would be fantastic, great addition :)

File: public/js/editors/libraries.js [hunk 1]
```diff
@@ -681,6 +681,16 @@ var libraries = [
     'url': 'https://cdnjs.cloudflare.com/ajax/libs/mori/0.3.2/mori.js',
     'label': 'mori 0.3.2',
     'group': 'Data structures'
+  },
+  {
+    'url': 'http://cdn.jsdelivr.net/ramda/0.15.1/ramda.min.js',
+    'label': 'Ramda 0.15.1',
+    'group': 'Ramda'
```
Comment (remy) [inline public/js/editors/libraries.js]: Does this really need a grouping? Groupings tend to be when there's an extremely high usage and need fo different versions (like jQuery) or there's different modules to be pulled in.

File: public/js/editors/libraries.js [hunk 2]
```diff
@@ -681,6 +681,16 @@ var libraries = [
     'url': 'https://cdnjs.cloudflare.com/ajax/libs/mori/0.3.2/mori.js',
     'label': 'mori 0.3.2',
     'group': 'Data structures'
+  },
+  {
+    'url': 'http://cdn.jsdelivr.net/ramda/0.15.1/ramda.min.js',
```
Comment (remy) [inline public/js/editors/libraries.js]: All CDN URLs should be served from https when available, as more and more users are switching to https by default, and this won't load.

**Comment by johnmichel** [comment]
@remy, I've made the suggested changes.

**Pull request merged**